### ISSUE TITLE
Show CFDA programs just for selected agency

### ISSUE THREAD

**Issue by kaitlin** [fedspendingtransparency/USASpending-Issue-Tracker#10]
When selecting an agency in the advanced search, I would recommend limiting the programs in the CFDA program dropdown to just show the programs relevant for that agency. This would make them easier to browse.

**Comment by swcaldwell** [comment]
Yes, this is an excellent suggestion, would love to see it implemented.
